lib/core/services/sort-by-category.service.spec.ts

```diff
@@ -0,0 +1,133 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { setupTestBed } from '../testing/setup-test-bed';
+import { CoreTestingModule } from '../testing/core.testing.module';
+import { TranslateModule } from '@ngx-translate/core';
+import { SortbleByCategoryItem, SortByCategoryMapperService } from './sort-by-category.service';
+
+interface TestSortbleByCategoryItem extends SortbleByCategoryItem {
+    id: string;
+}
+
+describe('SortByCategoryService', () => {
+
+    let mapper: SortByCategoryMapperService<TestSortbleByCategoryItem>;
+
+    const DEFAULT_CATEGORIES = [
+        '',
+        'http://bpmn.io/schema/bpmn',
+        'http://www.activiti.org/processdef',
+        'http://www.activiti.org/test'
+      ];
+
+    setupTestBed({
+        imports: [
+            TranslateModule.forRoot(),
+            CoreTestingModule
+        ]
+    });
```

Comment:
I think `setupTestBed` is not needed and can be removed

Reply:
Removed